### ISSUE TITLE
`supermin` augenrules

### ISSUE THREAD

**Issue by bphd** [linux-audit/audit-userspace#290]
**NOTE:** Please refer to the [Reporting Bug and Requesting Features](https://github.com/linux-audit/audit-documentation/wiki/Reporting-Bugs-and-Requesting-Features) wiki page before creating any new GitHub issues.

https://github.com/linux-audit/audit-userspace/blob/22c164293d961bad920ea70c4f16d6208bd0b853/init.d/Makefile.am#L68
https://github.com/linux-audit/audit-userspace/commit/8441313bcb82288d7caa6c622a19d2b3347cc909#diff-e3bd1114a2959a0cb463f24061c706b9994a23bb79dafd361828c5d3c857bef5
https://github.com/libguestfs/supermin/blob/f5a3deef588759c49dc862427a265aced612002d/src/ext2fs-c.c#L530-L539
https://github.com/libguestfs/supermin/commit/f5a3deef588759c49dc862427a265aced612002d#diff-43a61855ffa4b25027dacc23718fb03fa0160f4f49b3e080b4938382e9c2ee45
@sgrubb @stevegrubb @libguestfs @rwmjones (@ptoscano @hillu @olafhering)

> Some distro files are not public readable, so supermin cannot copy them
into the appliance.  This is a problem with your Linux distro.  Please ask
your distro to stop doing pointless security by obscurity.
You can ignore these warnings.  You *do not* need to use sudo.

**Comment by stevegrubb** [comment]
There are some files that are not publicly readable for common criteria purposes.

**Comment by bphd** [comment]
> @stevegrubb:
> common criteria purposes

> @rwmjones:
> stop doing pointless security by obscurity

**Comment by hillu** [comment]
Why was I tagged in this issue? And what is the actual problem?

**Comment by rwmjones** [comment]
> Why was I tagged in this issue? And what is the actual problem?

I think it's something to do with [this message in supermin](https://github.com/libguestfs/supermin/blob/fb469f42915f920b4a369f1520f8e7d872fc93bf/src/ext2fs-c.c#L494).  However I'd also like to know what the actual problem is, otherwise please don't tag me (or hillu).

**Comment by stevegrubb** [comment]
Closing this. The permissions in the make file are not the final permissions. The packaging system can overide it. Review of the Fedora spec file shows permissions are 0755, so I modified it. And no, it is not security by obscurity. The purpose is that if a common user can even attempt to run the utilities related to the audit trail, it adds a lot more monitoring to detect who is trying to alter the audit trail.
